docker-compose.yml

```diff
@@ -2,9 +2,9 @@ version: "3"
 services:
   prometheus:
     depends_on:
-      - kt-server
-      - kt-signer
-    image: us.gcr.io/key-transparency/prometheus
+      - server
+      - sequencer
+    image: us.gcr.io/key-transparency/prometheus:latest
```

Comment:
Isn't the "latest" tag used by default?

Reply:
It is, but there's a strange issue..
Kubernetes will only pull the latest image if `latest` is specified but I had difficulty getting Kubernetes to pull a fresh image without the `:latest` tag.

Reply:
Ah that could be because you aren't specifying an ImagePullPolicy (https://kubernetes.io/docs/concepts/configuration/overview/#container-images).